apps/script/convert-scss-variables.js

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env node
+
+/**
+ * Script that processes an scss file containing a set of variables, and
+ * generates the equivalent set in JS.
+ */
+var fs = require('fs');
+var path = require('path');
+var readline = require('readline');
+
+// Regular expression to capture a color definition from SCSS
+var variableRe = /\$([\w-]+)\s*:\s*([^;]+);/;
+var cachedVariables = {};
+
+// In color.scss some color definitions reference previous
+// color definitions.  Since we're reading them in order,
+// we can recursively resolve these referential defintions
+// until we land on an actual color.
+function resolveVariable(value) {
+  var originalValue = value;
+  while (/^\$/.test(value)) {
+    value = cachedVariables[value.substring(1)];
+  }
+  return value;
+}
+
+function convertScssToJs(scssPath, jsPath) {
+  // Generate <foo>.js while reading <foo>.scss line-by-line
+  var out = fs.createWriteStream(jsPath);
+  out.write([
+    '// color.js',
```

Comment:
Should adjust the filename in the header comment according to the `jsPath` argument.